src/test/rustdoc-json/primitive.rs

```diff
@@ -0,0 +1,9 @@
+// edition:2018
+
+#![feature(doc_primitive)]
+#[doc(primitive = "char")]
+pub mod char {}
```

Comment:
Primitive modules in the standard library aren't public. Can you make this private and make sure the test still passes?
```suggestion
mod char {}
```